src/App.tsx

```diff
@@ -10,33 +12,22 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState<Tab>(tabs[0]);
+
+  const onTabSelected = (chosenTab: Tab) => {
```

Comment:
```suggestion
  const handleTabSelected = (chosenTab: Tab) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364